src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,15 @@
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+import MovieCard from '../MovieCard/MovieCard';
+
+const MoviesList = ({ movies }) => (
+  <div className="page-content">
+    {movies.map(movie => (
+      <div className="movies" key={movie.imdbId}>
```

Comment:
There should be only one element with the `movies class. Move the map() logic inside this div.
![image](https://github.com/mate-academy/react_movies-list-js/assets/99403198/6534a43b-0ae6-4d8d-9747-3e9388add16c)